packages/shared/__tests__/ReactSymbols-test.internal.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+'use strict';
+
+describe('ReactSymbols', () => {
+  beforeEach(() => jest.resetModules());
+
+  const expectToBeUnique = keyValuePairs => {
+    const map = new Map();
```

Comment:
Maybe would be easier to get an array with `Object.values()` and then verify its length is the same as `new Set(arr).size`?

Reply:
Yeah, that would be easier but this provides a more meaningful error message if it fails. 😄